python/rmm/_lib/memory_resource.pyx

```diff
@@ -233,8 +249,16 @@ cdef class CudaAsyncMemoryResource(DeviceMemoryResource):
         Release threshold in bytes. If the pool size grows beyond this
         value, unused memory held by the pool will be released at the
         next synchronization point.
+    enable_ipc: Boolean, optional
+        If True, enables export of Posix file descriptor handle for the memory
+        allocated by this resource so that it can be used with CUDA IPC.
     """
-    def __cinit__(self, initial_pool_size=None, release_threshold=None):
+    def __cinit__(
+            self,
+            initial_pool_size=None,
+            release_threshold=None,
+            enable_ipc=False
+    ):
```

Comment:
(Since we don't have a way to autoformat our Cython code, this will have to do.)
```suggestion
    def __cinit__(
        self,
        initial_pool_size=None,
        release_threshold=None,
        enable_ipc=False
    ):
```